pages/_app.tsx

```diff
@@ -1,4 +1,5 @@
 import "@/styles/globals.css";
+import "@/styles/index.css";
```

Comment:
why not ibutang nalang sa globals css if ever i import rasad sa _app HAHAHAHA, pero oks rasad bitaw wa lang

Reply:
kay sa index raman siya ginagamit? hahahaha

Reply:
pero basically ang class kay ma gamit na sa every page hehehe